apple/internal/apple_xcframework_import.bzl

```diff
@@ -374,6 +400,9 @@ def _get_xcframework_library_with_xcframework_processor(
 
     return struct(
         binary = binary,
+        # TODO(asky)
+        debug_info_binaries = [],
```

Comment:
i don't actually think this can be implemented as-is, since the outputs are opaque